src/databricks/labs/remorph/install.py

```diff
@@ -43,6 +47,7 @@ def __init__(
         product_info: ProductInfo,
         resource_configurator: ResourceConfigurator,
         workspace_installation: WorkspaceInstallation,
+        upgrades: Upgrades,
```

Comment:
add localised pylint disable comment here, i approve it for this specific case. 

other question is why we need the `ResourceConfigurator`, but it's out of scope of this PR.